CHANGELOG.md

```diff
@@ -165,6 +165,7 @@ Released with 1.0.0-beta.37 code base.
 - Add missing subscription.on('connected') TS type definition (#3319)
 - Add missing bignumber.js dependency for TS types (#3386)
 - Upgrade swarm-js to 0.1.40 to remove npm vulnerability warning (#3399)
+- Fixed misleading error thrown on XMLHTTP request error
```

Comment:
This should be moved to the `UNRELEASED` section

Reply:
oki